dev/contracts/deploy-ephemeral

```diff
@@ -6,6 +6,6 @@ source dev/contracts/.env
 cd ./contracts
 
 # Update depencencies
-forge soldeer update
+forge soldeer install
```

Comment:
@fbac I changed these from `update` to install, so that we don't update the lockfile. I believe OpenZeppelin released a bad patch version and update was failing to unzip it. Let me know if you want to change it back?